### ISSUE TITLE
Implement default sort order for grid lines on the Conditions/Positions pages

### ISSUE THREAD

**Issue by elena-zh** [gnosis/conditional-tokens-explorer#208]
Currently, the query, in the application, doesn't have a sort.
It would be nice to have a default sort of lines by the last created at the top of the grid.

Also, it would be to have a default sort of lines by the last created at the top of the grid on: 

- [x] Split position - select condition pop-up grid
- [x] Merge positions - select condition pop-up grid
- [x] Merge positions - select position pop-up grid (I'm not sure - to discuss)
- [x] Report payout - select condition pop-up grid
- [x] Redeem position - select condition pop-up grid
- [x] Redeem position - select position pop-up grid (I'm not sure - to discuss)

**Comment by mariano-aguero** [comment]
Issue related https://github.com/gnosis/hg-subgraph/issues/19

**Comment by mariano-aguero** [comment]
The condition sort should be fixed here https://github.com/gnosis/conditional-tokens-explorer/pull/346/files

**Comment by elena-zh** [comment]
The issue is verified. 
However, several related issues are still reproducible: 
- positions appear in the positions list only after page reloading (#328)
- sort order for the position/condition column is broken on the positions/conditions pages (#381)
